docs/how-to/websites-on-ipfs/gateway-redirects.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Gateway redirects
+description: What gateway redirects are and how to use them with a website on IPFS.
+---
+
+# THIS IS A DRAFT. DO NOT MERGE.
+
+# To do:
+1. How to redirect old URL to a new place (301 and 302 redirects)
+1. How to use it for PWA/SPA hosting (catch-all 200) 
+1. How to use it to provide custom `404 not found` pages (superceding what `ipfs-404.html` does - ideally not mention old way)
+
+# Gateway Redirects
+
+Gateway Redirects provide support for URL redirects and rewrites for websites hosted on Subdomain or DNSLink gateways. This feature enables support for single-page applications, and avoids link rot when moving to IPFS-backed hosting.
+
+Using Gateway Redirects, you can change the appearance of a URL, change where content is located without breaking existing links, redirect invalid URLs to a custom 404 page, and enable URL rewriting.
+
+# Supported HTTP status codes
+
+* `200` - OK (redirect will be treated as a rewrite, returning OK without changing the URL shown in the browser).
+* `301` - Permanent redirect (the default status).
+* `302` - Found (commonly used for temporary redirects).
+* `303` - See other (replaces PUT and POST with GET).
+* `307` - Temporary redirect (preserves the body and HTTP method of the original request).
+* `308` - Permanent redirect (preserves the body and HTTP method of the original request).
+* `404` - Not found (can be used redirect to custom 404 pages).
+* `410` - Gone (the requested content has been permanently removed).
+* `451` - Unavailable for legal reasons.
+
+# How to set up gateway redirects
+
+To use Gateway Redirects, there must be a file named `_redirects` stored underneath the root CID of the website. This `_redirects` file must be a text file containing one or more lines that follow the format explained below.
```

Comment:
Would be good to preface with a paragraph explaining `_redirects` is a subset of preexisting convention/standard supported by [Cloudflare (docs)](https://developers.cloudflare.com/pages/platform/redirects) and [Netlify (docs)](https://docs.netlify.com/routing/redirects/). This builds trust a bit, as the reader now knows it is not something we just made up.